### PR TITLE
Validation: descriptions

### PR DESCRIPTION
**Pull request by SueADarby**
branch setup

### CODE REVIEW

File: iis/extensions/smooth-streaming-client/smoothstreamingmediaelement-smoothstreamingcacheproperty-field-microsoft-web-media-smoothstreaming_1.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: SmoothStreamingMediaElement.SmoothStreamingCacheProperty Field (Microsoft.Web.Media.SmoothStreaming)
+description: This page details the syntax for the SmoothStreamingMediaElement.SmoothStreamingCacheProperty Field. It represents a dependency property that represents the SmoothStreamingCache property.
```
Comment (dereklh7) [inline iis/extensions/smooth-streaming-client/smoothstreamingmediaelement-smoothstreamingcacheproperty-field-microsoft-web-media-smoothstreaming_1.md:3]: Description too long per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main

File: iis/extensions/transform-manager/loglevelhelper-getloglevel-method-string-microsoft-web-media-transformmanager.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 ---
 title: LogLevelHelper.GetLogLevel Method (String) (Microsoft.Web.Media.TransformManager)
+description: This page details the syntax for the GetLogLevel method and returns a value that specifies the level of detail for a log.
+
```
Comment (dereklh7) [inline iis/extensions/transform-manager/loglevelhelper-getloglevel-method-string-microsoft-web-media-transformmanager.md:4]: Remove empty line between description and TOCTitle. Empty lines in metadata can cause issues.

File: iis/extensions/express-api-reference/clearmanifestcontext.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 ---
 title: "ClearManifestContext"
+description: This page details the syntax for the ClearManifestContext. It clears the manifest context created using an earlier call to ApplyManifestContext.
```
Comment (dereklh7) [inline iis/extensions/express-api-reference/clearmanifestcontext.md:3]: A better description would be, "Describes ClearManifestContext and provides the manifest context's syntax, return value, and additional references."

File: iis/extensions/database-manager-reference/typeinfo-isvariablelength-property-microsoft-web-management-databasemanager.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 title: TypeInfo.IsVariableLength Property (Microsoft.Web.Management.DatabaseManager)
+description: This page details the syntax for the TypeInfo.IsVariableLength property which gets or sets a value that indicates whether the data type supports variable length.
```
Comment (dereklh7) [inline iis/extensions/database-manager-reference/typeinfo-isvariablelength-property-microsoft-web-management-databasemanager.md:3]: A tad too long per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.

File: iis/extensions/express-api-reference/iiisuserdata.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "IIISUserData"
+description: This page details the syntax for the IIISUserData. This allows per-user IIS Express directories to be queried and created for IIISVersion objects that describe IIS Express.
```
Comment (dereklh7) [inline iis/extensions/express-api-reference/iiisuserdata.md:3]: Too long per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.

File: iis/extensions/smooth-streaming-client/adaptivesourcemanager-sendextendedcommand-method.md [hunk 6]
```diff
@@ -1,5 +1,6 @@
 ---
 title: AdaptiveSourceManager.SendExtendedCommand Method
+description: This page details the syntax for the AdaptiveSourceManager.SendExtendedCommand method. It applies to Windows Store apps only. Reserved for future use.
```
Comment (dereklh7) [inline iis/extensions/smooth-streaming-client/adaptivesourcemanager-sendextendedcommand-method.md:3]: Reserved for future use? Is this a placeholder?

A better description would be, "Describes the AdaptiveSourceManager.SendExtendedCommand method and provides the method's syntax, parameters, and requirements."

File: iis/extensions/smooth-streaming-client/imanifesttrack-interface-maxwidth-property.md [hunk 7]
```diff
@@ -1,5 +1,6 @@
 ---
 title: IManifestTrack Interface.MaxWidth Property
+description: This page details the syntax for the IManifestTrack Interface.MaxWidth property and gets the track 'MaxWidth' field.
```
Comment (dereklh7) [inline iis/extensions/smooth-streaming-client/imanifesttrack-interface-maxwidth-property.md:3]: "Gets the track 'MaxWidth' field" is unclear - a better description would be "Describes the IManifestTrack Interface.MaxWidth property and provides the property's syntax, property value, and requirements."

File: iis/extensions/smooth-streaming-client/smoothstreamingasyncresult-completedsynchronously-property-microsoft-web-media-smoothstreaming.md [hunk 8]
```diff
@@ -1,5 +1,6 @@
 ---
 title: SmoothStreamingAsyncResult.CompletedSynchronously Property (Microsoft.Web.Media.SmoothStreaming)
+description: This page details the syntax for the SmoothStreamingAsyncResult.CompletedSynchronously property. This indicates whether the operation completed synchronously on the same thread.
```
Comment (dereklh7) [inline iis/extensions/smooth-streaming-client/smoothstreamingasyncresult-completedsynchronously-property-microsoft-web-media-smoothstreaming.md:3]: Too long per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.

File: iis/extensions/smooth-streaming-client/smoothstreamingmediaelement-naturalduration-property-microsoft-web-media-smoothstreaming_1.md [hunk 9]
```diff
@@ -1,5 +1,6 @@
 ---
 title: SmoothStreamingMediaElement.NaturalDuration Property (Microsoft.Web.Media.SmoothStreaming)
+description: This page details the syntax for the SmoothStreamingMediaElement.NaturalDuration property. This property gets the duration of the current stream when it plays to the end.
```
Comment (dereklh7) [inline iis/extensions/smooth-streaming-client/smoothstreamingmediaelement-naturalduration-property-microsoft-web-media-smoothstreaming_1.md:3]: Too long per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.

**Comment by SueADarby** [comment]
**Global effort to fix build validation errors**
 @Rick-Anderson  and @mcleblanc     
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**